src/Simplex/Chat/Messages/CIContent.hs

```diff
@@ -311,6 +312,7 @@ rcvGroupEventToText = \case
   RGEInvitedViaGroupLink -> "invited via your group link"
   RGEMemberCreatedContact -> "started direct connection with you"
   RGEMemberProfileUpdated {} -> "updated profile"
+  RGESkippedMessages -> "notified about skipped messages"
```

Comment:
draft - probably should have different text, e.g. "continued message delivery [after your inactivity]". and/or more info?

Reply:
maybe no item is required